Testscripts/Linux/SetupRDMA.sh

```diff
@@ -191,8 +191,8 @@ function Main() {
 			LogMsg "Starting RDMA setup for Ubuntu"
 			hpcx_ver="ubuntu"$VERSION_ID
 			LogMsg "Installing required packages ..."
-			LogMsg "*** Adding Canonical ppa for temporary fix"
-			add-apt-repository -y ppa:ci-train-ppa-service/3760
+			# the old fix integrated to dpdk-18.11 repo
+			add-apt-repository ppa:canonical-server/dpdk-azure-18.11 -y
```

Comment:
We'd better to have a check here to make sure the ppa repo is added successfully, like this time, we need do much investigation if not catch this error earlier from log.

Reply:
fixed in ad6635b367739d7217fcb9c837f5d20730c3f85b